Sources/App/Models/Package.swift

```diff
@@ -45,6 +51,27 @@ final class Package: Model, Content {
 }
 
 
+extension Package {
+    var repository: Repository? {
+        repositories.first
+    }
+}
+
+
+extension Package {
+    var defaultVersion: Version? {
```

Comment:
I'm not quite sure what `defaultVersion` is doing here? Are you referring to the git version tags here? If so, I don't think there should be a "default" version. Instead, we care about (up to) three versions.

1. **Latest**: The latest commit to the default branch.
2. **Latest Stable**: A tagged version that is a valid 3-part semantic version and does *not* have a "pre" part. For example 1.0.0.
3. **Latest Beta**: A tagged version that is a valid 3-part semantic version, and *does* have a "pre" part. For example 1.0.0-beta1.

~~If there is a "default" version it should be the latest version, but really I don't think there should be a default version. Happy to discuss what this is for and why it's needed.~~

Reply:
Scratch my last sentence above. If there is a default version it should be the **Latest Stable** as that's what Xcode chooses if you don't specify a version when adding a package.

Reply:
Yes, that makes sense!

I think we're on the same page here - the whole version stuff will change quite a bit still tbh and I don't think this is even used anywhere yet.

I'd probably take it out for now - or I'll add a TODO and clean it up later. Let me know!

Reply:
No need to take it out, let's just add a note around it?